server/service/GithubService.js

```diff
@@ -225,6 +224,7 @@ export default class GithubService {
       const pages = await Promise.all(pageDefs.map(async (page) => await that.fetchRepoPage(page, accessToken)))
       pages.forEach(p => Array.prototype.push.apply(repos, p.body))
     }
+    info('Loaded %d repos from Github', repos.length)
```

Comment:
Would log this as debug as it will be logged vor every user syncing github repos with upstream.